src/graph/backend/dnnl/dnnl_backend.hpp

```diff
@@ -111,24 +111,25 @@ class dnnl_backend_t : public backend_t {
         auto &pass_registry = get_pass_registry();
         graph::pass::pass_manager_t pm(pass_registry);
 
-#ifdef DNNL_ENABLE_GRAPH_DUMP
-        std::string pass_config_json = "dnnl_graph_passes.json";
-        std::ifstream fs(pass_config_json.c_str());
-        if (fs) {
-            verbose_printf(
-                    "graph,info,pattern,load,%s\n", pass_config_json.c_str());
-        } else {
-            if (getenv_int_user("GRAPH_DUMP", 0) > 0
-                    || graph::utils::check_verbose_string_user(
-                            "GRAPH_DUMP", "pattern")) {
+#ifdef DNNL_DISABLE_GRAPH_DUMP
```

Comment:
Given that `get_graph_dump_mode` is available at all times, why introducing extra macros at upper levels?

The idea behind a build time variable is to limit control over the application through env variables (which shouldn't be respected if it was opt-out) but not through a programmable way (which is to be added?).

Reply:
Hi @dzarukin , the reason I introduced this macro is that I want to make it align with DISABLE_VERBOSE, which also completely removes the code at build time. Are you suggesting the macro is unnecessary at all places or just here?

Reply:
I found that saving negligible amount of space not worth the maintain effort for the `defined(DNNL_DISABLE_GRAPH_DUMP)` case. BTW, have you tried to use it in a build? Is it successful so far?